### DISCUSSION TITLE
Documentation for Headless Blazor

### DISCUSSION THREAD

**Discussion by techbud** [OrchardCMS/OrchardCore#17212] [Q&A]
Hello, I'm new to Orchard Core CMS. I'm looking for Documentation on contentmanager, or how to pull in all my blog posts. the demo shows how to get it by its Alias. But I want to pull in all by its type. I thought ContentManager.GetAllVerionsAsync("BlogPost") would do it, but it didn't.

Can someone one point me to the documentation on how to do things the headless way? Thanks in advance.

**Comment by hishamco** [comment]
You can try `GetRecentContentItemsByContentType()` extension method on `IOrchardHelper`

**Comment by techbud** [comment]
I inject the class and try to use the method, but that method isn't showing as an option.

**Comment by techbud** [comment]
I'm watching and following along with this video:

https://www.youtube.com/watch?v=yWpz8p-oaKg

And some of the functions work, like CultureName and things like that but GetContentItemByAliasAsync isn't an option that I have.

**Comment by techbud** [comment]
Alright I figured out that you had to add the target nuget package to the Blazor Library project.
